tests/test_events.py

```diff
@@ -83,17 +83,16 @@ def test_can_manually_upload_agenda(event, admin_client, mocker):
     assert not event.documents.filter(note__icontains='Manual')
 
     with open('tests/test_agenda.pdf', 'rb') as agenda:
-        agenda_file = agenda.read()
-
-    agenda_pdf_form = AgendaPdfForm({'agenda': SimpleUploadedFile('test_agenda.pdf', agenda_file, content_type='application/pdf')})
-
-    # assert agenda_pdf_form.is_valid()
-
-    data = {'pdf_form': agenda_pdf_form, 'agenda': agenda_file}
-    admin_client.post(reverse("lametro:events", args=[event.slug]), data=data)
-
+        agenda_file = SimpleUploadedFile(
+            "test_agenda.pdf",
+            agenda.read(),
+            content_type="application/pdf"
+        )
 
-    event.refresh_from_db()
+        admin_client.post(
+            reverse("lametro:events", args=[event.slug]),
+            data={"pdf_form": "", "agenda": agenda_file},
```

Comment:
Passing the file as an instance of `SimpleUploadedFile`, rather than passing `agenda.read()`, is what fixed this! Also, didn't need to instantiate the form in the test. When I tested the upload manually, the `pdf_form` value is just an empty string.